src/System.Security.Cryptography.Xml/src/System/Security/Cryptography/Xml/Utils.cs

```diff
@@ -754,7 +754,11 @@ internal static bool IsSelfSigned(X509Chain chain)
 
         internal static AsymmetricAlgorithm GetAnyPublicKey(X509Certificate2 certificate)
         {
-            return (AsymmetricAlgorithm)certificate.GetRSAPublicKey() ?? certificate.GetDSAPublicKey();
+            return (AsymmetricAlgorithm)certificate.GetRSAPublicKey()
+#if netcoreapp || uap || uapaot
```

Comment:
You would need to add it to the props file. However you should be sure you need to add all those configurations just for this one difference. If the difference is really important then you should probably just not have a netstandard configuration.